crates/utils/src/settings/structs.rs

```diff
@@ -104,6 +104,24 @@ pub struct PictrsConfig {
   /// Resize post thumbnails to this maximum width/height.
   #[default(512)]
   pub max_thumbnail_size: u32,
+
+  /// Maximum size for user avatar, community icon and site icon.
+  #[default(512)]
+  pub max_avatar_size: u32,
+
+  /// Maximum size for user, community and site banner.
+  ///
+  /// TODO: Unfortunately pictrs can only resize images to fit in a*a square, no rectangle.
+  ///       Otherwise we have to use crop, or use max_width/max_height which throws error
+  ///       if image is larger.
+  /// TODO: whats a sensible default here?
+  #[default(512)]
```

Comment:
Banners should never be square, so we probably should do max width + max height.

I found some banner sizes for current websites [here](https://www.template.net/graphic-design/banner-sizes/).